app/views/shared/_errors.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if local_assigns.fetch(:model, nil) && local_assigns[:model].errors.any? %>
```

Comment:
I think local_assigns is just a `Hash` (although, knowing Rails, could be anything, hehe)